modules/fbx/fbx_document.cpp

```diff
@@ -1076,13 +1075,15 @@ Error FBXDocument::_parse_images(Ref<FBXState> p_state, const String &p_base_pat
 			memcpy(data.ptrw(), fbx_texture_file.content.data, fbx_texture_file.content.size);
 		} else {
 			String base_dir = p_state->get_base_path();
-			Ref<Texture2D> texture = ResourceLoader::load(_get_texture_path(base_dir, path), "Texture2D");
-			if (texture.is_valid()) {
-				p_state->images.push_back(texture);
-				p_state->source_images.push_back(texture->get_image());
-				continue;
+			if (!FileAccess::exists(_get_texture_path(base_dir, path))) {
```

Comment:
This checks that the file *doesn't* exist and then tries to load it, how does that work?